src/list.tsx

```diff
@@ -173,18 +170,12 @@ const Content: React.FC<ContentProps> = ({
 
     return (
         <Box paddingY={16} className={classes.content}>
```

Comment:
I think that in figma they have specified a special alignment for the items in the row, depending on how many items are displayed in rowBody:
**from Figma:**
![image](https://user-images.githubusercontent.com/681159/130597192-f299b3dc-0dba-4a32-b063-4f3bcde77fbe.png)

**Our result:**
![image](https://user-images.githubusercontent.com/681159/130597433-62678e12-7668-42e6-b73c-f09df301504b.png)

Should we review this with @yceballost ??

Reply:
![image](https://user-images.githubusercontent.com/579417/130600384-5ad0928f-55bf-484c-a131-26b9859cfe72.png)
It was an error in spec definition. Specs have been updated

Reply:
solved: fixed in specifications